client/me/connected-application-item/index.jsx

```diff
@@ -8,11 +8,12 @@ var React = require( 'react' ),
 /**
  * Internal dependencies
  */
-var eventRecorder = require( 'me/event-recorder' ),
-	CompactCard = require( 'components/card/compact' ),
-	ConnectedApplicationIcon = require( 'me/connected-application-icon' ),
-	safeProtocolUrl = require( 'lib/safe-protocol-url' ),
-	analytics = require( 'analytics' );
+import eventRecorder from 'me/event-recorder';
+import ConnectedApplicationIcon from 'me/connected-application-icon';
+import safeProtocolUrl from 'lib/safe-protocol-url';
+import analytics from 'analytics';
+import Button from 'components/button';
+import FoldableCard from 'components/foldable-card';
 
 module.exports = React.createClass( {
```

Comment:
Since we are already rewriting the import statements, we could/should go ahead and rewrite the export too methinks

```js
export default React.createClass( {
	…
} )
```